clients/drcachesim/tracer/tracer.cpp

```diff
@@ -169,6 +169,17 @@ clean_call(void)
     process_and_output_buffer(drcontext, false);
 }
 
+void
+instru_notify(uint level, const char *fmt, ...)
```

Comment:
Curious why this couldn't just be a part of instru_offline.cpp

Reply:
Today instru_offline.cpp has some nice isolation from the rest of the tracer: it does not include the options.h or utils.h and is a pure library that does not share any of the configuration or reporting infrastructure specific to the tracer.  It seemed nice to preserve that.  It is used by raw2trace as well and raw2trace consequently does not have to pull in the tracer's options/logging code and deal with it or have to stub it out like we have to do with libraries we've made from the core because we don't have great isolation everywhere (xref those weak symbol complexities).